package proto_ff;

import "yd_fieldoptions.proto";


message E_TitleType
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "称号id"];
}


message Sheet_TitleType
{
	repeated E_TitleType E_TitleType_List = 1[(yd_fieldoptions.field_arysize)=8];
}

message E_TitleTitleAttributeDesc
{
	optional int32 m_value = 1[(yd_fieldoptions.field_cname) = "值"];
	optional int32 m_type = 2[(yd_fieldoptions.field_cname) = "类型"];
}

message E_TitleTitle
{
	optional int64 m_id = 1[(yd_fieldoptions.field_cname) = "称号ID"];
	optional string m_name = 2[(yd_fieldoptions.field_cname) = "称号名称", (yd_fieldoptions.field_bufsize) = 64];
	optional int32 m_type = 3[(yd_fieldoptions.field_cname) = "称号类型"];
	optional int32 m_subType = 4[(yd_fieldoptions.field_cname) = "获取条件类型"];
	optional string m_eventValue = 5[(yd_fieldoptions.field_cname) = "获取条件值", (yd_fieldoptions.field_bufsize) = 64];
	optional string m_describe = 6[(yd_fieldoptions.field_cname) = "获取条件描述", (yd_fieldoptions.field_bufsize) = 256];
	optional int32 m_addTime = 7[(yd_fieldoptions.field_cname) = "自动延期"];
	optional int32 m_timeTpye = 8[(yd_fieldoptions.field_cname) = "时间类型"];
	optional int32 m_lifeTime = 9[(yd_fieldoptions.field_cname) = "生效时间"];
	optional int64 m_activationItem = 10[(yd_fieldoptions.field_cname) = "激活道具ID"];
	optional int32 m_activationNum = 11[(yd_fieldoptions.field_cname) = "激活道具数量"];
	optional int64 m_starId = 12[(yd_fieldoptions.field_cname) = "升星道具id"];
	optional string m_starNum = 13[(yd_fieldoptions.field_cname) = "升星道具数量", (yd_fieldoptions.field_bufsize) = 256];
	optional int32 m_starUp = 14[(yd_fieldoptions.field_cname) = "升星等级上限"];
	optional int32 m_starBer = 15[(yd_fieldoptions.field_cname) = "每次升星属性增加的万分比值"];
	repeated E_TitleTitleAttributeDesc m_attribute = 16[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 6];
}


message Sheet_TitleTitle
{
	repeated E_TitleTitle E_TitleTitle_List = 1[(yd_fieldoptions.field_arysize)=128];
}
